frontend/src/contexts/ContextProvider.js

```diff
@@ -10,7 +10,7 @@ const StateContext = createContext({
 
 export const ContextProvider=({children})=>{
     const [user,setUser]=useState({});
-    const [token,_setToken]=useState(localStorage.getItem('ACCESS_TOKEN')); /*localStorage.getItem('ACCESS_TOKEN')*/
+    const [token,_setToken]=useState(1); /*localStorage.getItem('ACCESS_TOKEN')*/
```

Comment:
```suggestion
    const [token,_setToken]=useState(localStorage.getItem('ACCESS_TOKEN'));
```

Reply:
Zgaduje że podmieniłeś sobie żeby mieć szybciej dostęp ale jak mamy wrzucić na maina to wolał bym prawidłową wersje useState(localStorage.getItem('ACCESS_TOKEN'));

Reply:
Zmieniliśmy na poprawną wersje